contracts/WandName.sol

```diff
@@ -0,0 +1,18 @@
+  string[] adjectives = ["dazzling","mirrorlike","cloudy","nontranslucent","nontransparent","coarse","nubilous","obscure","moist","dense","leaden","gleaming","opaque","glistening","glittering","moonlit","humid","dark","igneous","fierce","misty","phosphorescent","mucky","damp","Fiery","flaming","flaring","murky","fervid","flashing","dark","aglow","dim","light","fluid","alight","lighted","aqueous","foggy","relucent","auroral","bawdy","gloomy","resplendent","burnished","beaming","robust","dewy","glossy","fulgent","funky","illuminated","enveloped","illumined","glowing","incandescent","luminous","lustrous","mushy","shimmering","dusky","ablaze","feverish","shiny","indefinite","marshy","shrouded","enthusiastic","emulsified","natural","inflamed","intense","fuzzy","golden","nebulous","bleary","glaring","clouded","burning","blurred","silvery","hazy","febrile","brilliant","blazing","heated","heavy","hot","overcast","polished","radiant","rough","somber","sparkling","spirited","sullen","sunless","sunlit","sunny","twinkling","vaporous","vivid","waterlike"];
+  string[] nouns = ["obscurity","heavens","band","billow","globe","bowl","wandering star","celestial sphere","rack","orb","vault","colure","circlet","compass","island universe","revolution","empyrean","mist","universe","star system","heavens","world","ring","meridian","ether","entry","nature","perimeter","frost","opening","Milky Way","equator","orbit","smog","gloom","gate","spiral galaxy","macrocosm","irregular galaxy","cloud","overcast","pother","lid","hoop","entrance","pressure","coil","periphery","puff","scud","star cluster","fog","asteroid","fogginess","galaxy","horoscope","vault of heaven","luminous body","veil","cosmos","steam","creation","aureole","cycle","murk","earth","darkness","planetoid","halo","astrometry","sphere","vapor","disk","sky","azure","smoke","microcosm","film","crown","star","doorway","haze","disc","belt","round","terrene","dimness","elliptical galaxy","wheel","circuit","cirque","corona","ecliptic","enclosure","full turn","gateway","haziness","heavenly body","horizon","nebula","air"];
+
+  function generateWandName(uint randomSeed) public returns (string memory wandName) {
+    string memory actionAdjective = actionAdjectives[actionAdjectives.length * randomSeed % 10];
```

Comment:
mod 10 will only capture words with index 0-9, so none of the words past index 9 can be possibly selected. Do we mean to mod by the length of array to capture everything?

Reply:
Yup, not sure what I was thinking here.